packages/firestore/test/integration/api/query.test.ts

```diff
@@ -800,6 +813,14 @@ apiDescribe('Queries', (persistence: boolean) => {
       // With objects.
       const snapshot2 = await coll.where('zip', 'in', [{ code: 500 }]).get();
       expect(toDataArray(snapshot2)).to.deep.equal([{ zip: { code: 500 } }]);
+
+      // With null.
+      const snapshot3 = await coll.where('zip', 'in', [null]).get();
```

Comment:
Do we need a test that shows that "in" with null/nan and something else still finds the something else?

Reply:
done.